protocol-util/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.camunda</groupId>
+    <artifactId>zeebe-parent</artifactId>
+    <version>1.4.0-SNAPSHOT</version>
+    <relativePath>../parent/pom.xml</relativePath>
+  </parent>
+
+  <artifactId>zeebe-protocol-util</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Zeebe Protocol Utilities</name>
```

Comment:
I'm missing some documentation on what this module is intended for (I know I know, it's in the commit message). I think this is the type of info where we can go for more prominent documentation, like:

🔧 Please provide a description tag
🔧 Please add a README.md (can be short)

Reply:
Do you mean like the `protocol`'s README?

I quote:

> # Zeebe Protocol
>
> Binary protocol of Zeebe.

This is almost poetry :smile:

Reply:
So it would be something like this? 🤣 

> # Zeebe Protocol Util
> 
> Utilities for the binary protocol of Zeebe

Reply:
Added one